files.cc

```
@@ -35,28 +35,28 @@ static fd_t *first_fd = 0;
-	fd_t *thisfd;
-	/* Save info for debugging */
-	thisfd = new fd_t{};
-	thisfd->fd = fd;
-	thisfd->filename = file;
-	thisfd->flg = flg;
-	thisfd->pid = pid;
-	thisfd->next = first_fd;
-	first_fd = thisfd;
+    fd_t *thisfd;
+    /* Save info for debugging */
+    thisfd = new fd_t{};
```

Cppcheck (2.10-2)

[error] Uninitialized variables

(uninitvar)